src/WooCommerce.php

```diff
@@ -721,4 +721,26 @@ public static function getProductAttributes(\WC_Product $product) {
     return $data;
   }
 
+  /**
+   * Add salutation field for billing and shipping.
+   *
+   * @implements woocommerce_checkout_fields
+   */
+  public static function woocommerce_checkout_fields(?array $fields): array {
```

Comment:
I can't remember all details, but I'm fairly sure that this should use the hook `woocommerce_default_address_fields` as in https://github.com/netzstrategen/wordpress-ssofact/blob/master/src/WooCommerce.php#L302-L314

Reply:
This code is copied over from https://github.com/netzstrategen/woocommerce-moeve/blob/master/src/Plugin.php#L61 and works fine there.

Reply:
If I remember correctly, it will not appear in the administrative user profile or somewhere else this way. But okay, I guess this can still be refactored later, if necessary.